.gitattributes

```diff
@@ -1,3 +1,4 @@
 *.png filter=lfs diff=lfs merge=lfs -text
+*.mcap filter=lfs diff=lfs merge=lfs -text
 testdata/mcap/demo.mcap filter=lfs diff=lfs merge=lfs -text
 testdata/ filter=lfs diff=lfs merge=lfs -text
```

Comment:
Are we planning to keep `testdata` its confusing to have both `tests/` and `testdata/` at the top level.

Reply:
I agree and think we should remove it, just haven't looked into what it is used for right now.